tsc complains about bad import for `../../types/resources/annotation` in `annotation.d.ts`

```
../node_modules/.pnpm/@iiif+presentation-2@1.0.1_@iiif+presentation-3@1.0.6/node_modules/@iiif/presentation-2/resources/annotation.d.ts:6:34 - error TS2307: Cannot find module '../../types/resources/annotation' or its corresponding type declarations.

import { SpecificResource } from '../../types/resources/annotation';
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
````
https://github.com/IIIF-Commons/presentation-2-types/blob/main/resources/annotation.d.ts#L6

It seems to me that `SpecificResource` is not included in the `@iiif/presentation-2` module, but is instead part of `@iiif/presentation-3`: https://github.com/IIIF-Commons/presentation-3-types/blob/b6a207ca7ae5d72651363c52e4111738f91dccb4/resources/annotation.d.ts

--- a/resources/annotation.d.ts
+++ b/resources/annotation.d.ts
@@ -1,9 +1,10 @@
+import { SpecificResource } from '@iiif/presentation-3';
+
 import { OmitProperties, OneOrMany } from '../utility';
 import { TechnicalProperties } from '../iiif/technical';
 import { DescriptiveProperties } from '../iiif/descriptive';
 import { LinkingProperties } from '../iiif/linking';
 import { ContentResource } from './content-resource';
-import { SpecificResource } from '../../types/resources/annotation';
 import { RightsProperties } from '../iiif/rights';
 
 type AnnotationOmittedTechnical = '@id' | 'format' | 'height' | 'width' | 'viewingDirection' | 'navDate';

